sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/redis/JedisRedisCheckpointStore.java

```diff
@@ -2,18 +2,36 @@
 // Licensed under the MIT License.
 package com.azure.messaging.eventhubs.checkpointstore.redis;
 
+import com.azure.core.util.logging.ClientLogger;
 import com.azure.messaging.eventhubs.CheckpointStore;
 import com.azure.messaging.eventhubs.models.Checkpoint;
 import com.azure.messaging.eventhubs.models.PartitionOwnership;
+import reactor.core.Exceptions;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
-
+import java.util.HashMap;
+import java.util.Map;
 import java.util.List;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+import redis.clients.jedis.Jedis;
 
 /**
  * Implementation of {@link CheckpointStore} that uses Azure Redis Cache, specifically Jedis.
  */
 public class JedisRedisCheckpointStore implements CheckpointStore {
+    private static final ClientLogger LOGGER = new ClientLogger(JedisRedisCheckpointStore.class);
+    private final RedisClientConfig config = RedisClientConfig.getInstance();
+    private JedisPoolConfig createPoolConfig() {
```

Comment:
nit: it may make your code easier to read/reason about if you consider a class structure like:.

1. member variables
    1. public
    2. private
1. constructors
    1. public
    2. private 
3. getters/setters
4. public methods
5. private methods